sklearn/feature_extraction/tests/test_dict_vectorizer.py

```diff
@@ -76,6 +76,23 @@ def test_one_of_k():
     assert "version" not in names
 
 
+def test_iterable_value():
+    D_in = [{"version": ["1", "2"], "ham": 2},
+            {"version": "2", "spam": .3},
+            {"version=3": True, "spam": -1}]
+    v = DictVectorizer()
+    X = v.fit_transform(D_in)
+    assert X.shape == (3, 5)
```

Comment:
We can explicitly check for what the values are.

Reply:
Done. As a side note, I think that those tests were written that way on purpose in order to keep consistency with the [previous one](https://github.com/scikit-learn/scikit-learn/blob/b5205fddbd6b2d106af7024ea4079c630a93e164/sklearn/feature_extraction/tests/test_dict_vectorizer.py#L63).